src/components/PassphraseDialog/PassphraseDialog.scss

```diff
@@ -143,6 +152,13 @@ $color-backlog-blue--dark: #5c8fff;
   }
 }
 
+.form__error-message {
+  grid-column-start: 2;
+  color: $color-planning-pink--light;
```

Comment:
I used it here because the design says so. But I do actually prefer it over the red we had before, it feels more in line with our overall design. And in this context I think it still achieves the desired result of drawing attention/showing that something is wrong, because it is close to red and the only pink element on the page.